packages/shared/locales/en.json

```diff
@@ -133,15 +136,52 @@
             "body1": "This is an automated process to transfer your funds to your new wallet. We will check for spent addresses that weaken your security.",
             "body2": "It may take some time. Do not turn off your device during the migration.",
             "existing": "Your old wallet",
-            "new": "Your new wallet"
+            "new": "Your new wallet",
+            "learn": "Learn about the migration",
+            "beginMigration": "Begin Migration",
+            "migrating": "Migrating funds...",
+            "migrated": "{balance} migrated",
+            "migrationFailed": "Migration Failed"
+        },
+        "bundleMiningWarning": {
+            "title": "Warning",
+            "body1": "You have funds detected on a spent address. ",
+            "body2": "These are at risk of theft.",
+            "body3": "We recommend you run an automated process to help secure and transfer your funds safely.",
+            "learn": "Learn about spent addresses"
+        },
+        "secureSpentAddresses": {
+            "title": "Secure spent addresses",
+            "body1": "You have funds detected on {number} spent address.",            
+            "body2": "We recommend you run an automated process to help secure and transfer your funds safely.",          
+            "error": "You can’t migrate spent addresses with less than 1 Mi"           
+        },
+        "securityCheckCompleted": {
+            "title": "Security check completed",
+            "body1": "We have calculated the security risk level. Running this process again may lower your risk level.",            
+            "rerun": "Rerun process for selected addresses",
+            "continueMigration": "Continue migration process"
+        },
+        "securingSpentAddresses": {
+            "title": "Securing spent addresses...",
+            "body1": "You have funds detected on a spent address.",
+            "body2": "These are at risk of theft.",
+            "body3": "We recommend you run this automated process to help secure and transfer your funds safely.",
+            "learn": "Learn about spent addresses"
```

Comment:
These seem like duplicates of the `bundleMiningWarning` strings

Reply:
Removed as they are not being used